lib/slack/ApiMock.js

```diff
@@ -53,7 +58,6 @@ module.exports.bindMockApi = function (controller) {
     controller.axiosMockAdapter = axiosMockAdapter;
     controller.apiLogByKey = apiLogByKey;
     controller.httpBodyLog = httpBodyLog;
-    controller.adapter.slack.axios = axios;
```

Comment:
I see we configure some axios logic above but you removed this line. It looks like we still use axios below.

1. Why was this line removed?
2. Can you confirm we still require axios in this file?

Reply:
yes we still use it, you can see below in `spawn`